python/pacemaker/_cts/tests/base.py

```diff
@@ -0,0 +1,915 @@
+""" Base classes for CTS tests """
+
+__all__ = ["CTSTest", "RemoteDriver", "SimulStartLite", "SimulStopLite", "StartTest", "StopTest"]
```

Comment:
Since we're doing a tests submodule, I'm thinking one source file per test class. It'll be a lot of files but very easy to tell what you're looking for. You can do that in separate commits so as not to have to separate out all the work already done.

Reply:
Added a new commit to split everything up.  I wanted to do it in separate commits for each, but there's really nothing interesting to it and I didn't want to add a bunch of intermediate importing code that I'd just get rid of in a later patch.

Reply:
I've left CTSTest in base.py, but I'm not thrilled.  I could rename that file to ctstest.py, or put it in `__init__.py` (which I am also not thrilled about).

Reply:
I though the convention was naming single-class files after the class (like CTSTest.py), but I guess there's no official recommendation

Reply:
Also moved what's left in base.py to ctstest.py.